horovod/torch/mpi_ops.py

```diff
@@ -22,19 +22,20 @@
 # Load all the necessary PyTorch C types.
 import torch
 
-from horovod.torch.compression import Compression
-try:
+if LooseVersion(torch.__version__) >= LooseVersion('0.4.2'):
```

Comment:
I believe we concluded that because there's no version 0.4.2, we should compare against 1.0.0 instead (see test_torch.py).

Reply:
Good point!  Updated.